docker/docker-entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 set -e
 
+U=${U:-$(id -u)}
```

Comment:
So, this is run within the docker container, right? If `U`/`G` are not defined, what will id -u/id -g be? Like the default docker user? Can you add an in-source comment with the answer, as I will likely forget. :-)